includes/Generator/Customer.php

```diff
@@ -73,13 +61,37 @@ public static function generate( $save = true ) {
 			default:
 				break;
 		}
+
```

Comment:
@masteradhoc While it may be a bit before we get to review this, the added code isn't formatted according to [WP Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/) which we would need before merging.